tailwind.config.js

```diff
@@ -42,18 +30,31 @@ module.exports = {
         active: '#E2E6EB',
         hovercancel: '#9B1A1B',
         cancel: '#9B1A1A',
+        customBlue: '#2D719D',
+        customYellow: '#E9C368',
+        customGreen: '#2E9E8F',
+        customGray: '#37475A',
+        customBlack: '#131A22',
+        backgroundImage: {
+          login: "",
+        },
+        colors: {
+          brand: "#37475A",
+        },
+        fontFamily: {
+          poppins: ["Poppins", "sans-serif"],
+        },
       },
-    },
-    screens: {
-      xs: { max: "425px" },
-      sm: { min: "426px" },
-      md: { min: "769px" },
-      mdl:{ min: "1024px"},
-      lg: { min: "1440px" },
-      xl: { min: "2560px" },
-      laptop: { max: "1165px"}
```

Comment:
why did you remove these tailwind screen configuration, some of us used them in responsiveness